src/Sylius/Behat/Context/Api/Admin/ManagingProductReviewsContext.php

```diff
@@ -0,0 +1,231 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Service\SharedStorage;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\ProductReviewTransitions;
+use Sylius\Component\Review\Model\ReviewInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingProductReviewsContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    /** @var SharedStorage */
+    private $sharedStorage;
+
+    public function __construct(
+        ApiClientInterface $client,
+        SharedStorageInterface $sharedStorage
+    ) {
+        $this->client = $client;
+        $this->sharedStorage = $sharedStorage;
+    }
+
+    /**
+     * @When I browse product reviews
+     * @When I want to browse product reviews
+     */
+    public function iWantToBrowseProductReviews(): void
+    {
+        $this->client->index('product_reviews');
+    }
+
+    /**
+     * @When I want to modify the :productReview product review
+     */
+    public function iWantToModifyTheProductReview(ReviewInterface $productReview): void
```

Comment:
Could be
```suggestion
    public function iWantToModifyTheProductReview(ProductReviewInterface $productReview): void
```

Reply:
I'm not sure because we don't have this interface :D
`ProductReview`` extends `Review` which implements `ReviewInterface` :)